mindjunkies/forums/models.py

```
@@ -34,78 +39,97 @@ def save(self, *args, **kwargs):
+    body = models.CharField(max_length=150)
+    likes = models.ManyToManyField(
+        settings.AUTH_USER_MODEL, related_name="likedreplies", through="LikedReply"
     )
-    created_at = models.DateTimeField(auto_now_add=True)
-    is_read = models.BooleanField(default=False)
+    created = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self):
+        try:
+            return f"{self.author.username} : {self.body[:30]}"
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)